mod_reforged/hooks/entity/tactical/enemies/alp.nut

```diff
@@ -5,12 +5,6 @@
 		local b = this.m.BaseProperties;
 		b.setValues(this.Const.Tactical.Actor.Alp);
 		b.Initiative += this.Math.rand(0, 55);
-		b.IsAffectedByNight = false;
```

Comment:
The convention we are going for with entity files is that you don't "delete" the lines you don't need, instead you comment them out. So, for all the lines you are deleting in these entities, please comment out these lines instead, and then put a comment that they are now applied via their racial skill.